lib/bulk/unordered.js

```diff
@@ -1,18 +1,16 @@
 'use strict';
 
 const common = require('./common');
+const BulkOperationBase = common.BulkOperationBase;
+const FindOperatorsBase = common.FindOperatorsBase;
```

Comment:
as far as I can tell, it looks like the only difference between `FindOperatorsOrdered` and `FindOperatorsUnordered` is that they have a method: `isOrdered` that returns either `true` or `false`. Instead of having a `FindOperatorsBase`, why not just have a single `FindOperators` that takes in an `ordered: Bool`, and returns whether its ordered or unordered?